Add new feature alert to dashboard

nabramow opened the pull request:
<!---
Please describe the pull request below.
If it closes an issue, make sure to write "closes #1234"
If there is an issue but it isn't completely closed, still refer to the issue number, eg. "part of #1234"
--->

Thought this would draw more attention to when we release new features worth noting to draw users' attention to them. 

<!---
Checklists - you can remove one that is not applicable (ie. remove backend checklist if you only worked on the web frontend)
If you need help with any of these, please ask :)
--->
**Web frontend checklist**
- [ x ] Formatted my code with `yarn format`
- [ x ] There are no warnings from `yarn lint --fix`
- [ x ] There are no console warnings when running the app
- [ ] Added tests where relevant
- [ ] All tests pass
- [ x ] Clicked around my changes running locally and it works
- [ x ] Checked Desktop, Mobile and Tablet screen sizes

<!---
Remember to request review from couchers-org/web, couchers-org/backend or an individual.
Once your code is approved, remember to merge it if you have write access
--->

nabramow committed:
Add new feature alert

aapeliv commented:
![Screenshot from 2025-02-22 22-35-13](https://github.com/user-attachments/assets/c3bfffb7-5df5-460b-b846-87f48a1deab0)

* There's a weird amount of padding inside the blue box under the text?
* Also, do you think it would take a lot to make it read the text from some kind of markdown file (or even just html) so it's much easier for devs to update it?

nabramow committed:
Move news to json

nabramow commented:
Okay @aapeliv I fixed the padding and moved the news to a json in the main web folder with a key for the date. I'm thinking we can add news with the date there as we go. Also makes it more clear the last date it was updated.

Let me know if you're happy with that concept!

aapeliv reviewed (approved):
LGTM!
